src/map.rs

```diff
@@ -1348,6 +1348,38 @@ where
         self.put(key, value, false, guard)
     }
 
+    #[inline]
```

Comment:
Nit: We recently cleaned up all `#[inline]` annotations to be directly above the method signature (below the comment) with https://github.com/jonhoo/flurry/commit/c326f681cd729ad01cc109f450610cbbd408f059. It would be nice if we kept that consistent now

Reply:
No problem, I'll fix this.